app/decorators/case/overturned_ico/sar_decorator.rb

```diff
@@ -5,6 +5,9 @@ def original_case_description
   end
 
   def type_abbreviation
+    # This string is used when constructing paths or methods in other parts of
+    # the system. Ensure that it does not come from a user-supplied parameter,
+    # and does not contain special chars like slashes, etc.
```

Comment:
Not quite sure why this comment is under all the type abbreviation methods.  It's hard coded for each class, so there's no danger of it being overwritten.

Reply:
It's a warning in case we ever decide to change it from being hard-coded. [Code climate brought this up](https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff/issues?status%5B%5D=&status%5B%5D=invalid&status%5B%5D=wontfix&engine_name%5B%5D=brakeman) in the views where `type_abbreviation` is used to generate paths to templates to render so as I've silenced the warning there I thought I'd better drop a note in to make sure we're careful about changing this.